content/app/how-tos/ingress-with-whitelist.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Create Ingress with IP Whitelisting"
+weight = 5
+chapter = false
+pre = ""
++++
+
+{{% notice warning %}}
+This uses beta features in the platform and breaking changes may occur in the future
+{{% /notice %}}
+
+## Create middleware
+{{% notice note %}}
+If on your resource you come across a `middleware.traefik.containo.us`, that is an older version of Treaefik's CRD. It will be deprecated in the future.
+{{% /notice %}}
+
+Create a `middleware.traefik.io` object to whitelist IP addresses:
+
+```yaml
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
```

Comment:
is there a link to some public docs for this? would be nice to put it in this file somwhere